hs-init.hs

```diff
@@ -49,19 +49,19 @@ import           System.Process        (callCommand, readProcess, showCommandFor
 -----------------------
 
 defaultOwner :: Text
-defaultOwner = "vrom911"
```

Comment:
Can we keep my data in the file and change it only if the users input theirs own default values?
I'm using `stack template` to create projects and the `{{smth}}` structure creates conflicts and the tool fails.. So I want some default valid data in this variables.

Reply:
Sure!